pkg/handler/project/project.go

```diff
@@ -226,3 +228,82 @@ func (h *handler) Create(c *gin.Context) {
 
 	c.JSON(http.StatusOK, view.CreateResponse(view.ToCreateProjectDataResponse(p), nil, done(nil), nil))
 }
+
+// GetStaffs godoc
+// @Summary Get list staffs of project
+// @Description Get list staffs of project
+// @Tags Project
+// @Accept  json
+// @Produce  json
+// @Param Authorization header string true "jwt token"
+// @Param id path string false "Project ID"
+// @Param status query string false "Status"
+// @Param page query string false "Page"
+// @Param size query string false "Size"
+// @Param sort query string false "Sort"
+// @Success 200 {object} view.ProjectMemberListResponse
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 404 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /projects/:id/staffs [get]
+func (h *handler) GetStaffs(c *gin.Context) {
+	query := GetListStaffInput{}
+	if err := c.ShouldBindQuery(&query); err != nil {
+		c.JSON(http.StatusBadRequest, view.CreateResponse[any](nil, nil, err, query))
+		return
+	}
+	query.Standardize()
+
+	projectID := c.Param("id")
+	if projectID == "" {
+		c.JSON(http.StatusBadRequest, view.CreateResponse[any](nil, nil, errors.New("invalid project_id"), nil))
+		return
+	}
+
+	// TODO: can we move this to middleware ?
+	l := h.logger.Fields(logger.Fields{
+		"handler":   "project",
+		"method":    "GetStaffs",
+		"projectID": projectID,
+		"query":     query,
+	})
+
+	if err := query.Validate(); err != nil {
+		l.Error(err, "validate failed")
+		c.JSON(http.StatusBadRequest, view.CreateResponse[any](nil, nil, err, query))
+		return
+	}
+
+	exist, err := h.store.Project.Exist(h.repo.DB(), projectID)
+	if err != nil {
+		l.Error(err, "failed to check project existence")
+		c.JSON(http.StatusInternalServerError, view.CreateResponse[any](nil, nil, err, query))
+		return
+	}
+
+	if !exist {
+		l.Info("project not found")
+		c.JSON(http.StatusNotFound, view.CreateResponse[any](nil, nil, errors.New("project not found"), nil))
```

Comment:
tạo 1 biến error mới em
errors.New("project not found")

Reply:
updated